### PR TITLE
fix(details): show meter selected details, when appears in the map.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#547, #564 

screen shot 2015-06-24 at 1 31 22 pm

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
Handle message for the request of an inexistent meter:
screen shot 2015-06-23 at 4 48 20 pm

File: client/app/scripts/controllers/details.js [hunk 1]
```diff
@@ -23,11 +23,32 @@ angular.module('negawattClientApp')
...
+      }
+    });
+
     /**
      * Electricity Service Event: When electricity change update Pie chart of
      * consumption, according the summary data.
      */
-    $scope.$on("nwElectricityChanged", function(event, electricity) {
+    $scope.$on('nwElectricityChanged', function(event, electricity) {
+      if(angular.isDefined($stateParams.markerId)) {
```
Comment (amitaibu) [inline client/app/scripts/controllers/details.js]: `if (`

File: client/app/scripts/controllers/dashboard.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
 'use strict';
 
 angular.module('negawattClientApp')
-  .controller('DashboardCtrl', function ($state, $stateParams, profile) {
-    var defaultAccountId;
+  .controller('DashboardCtrl', function ($scope, $timeout, $state, $stateParams, profile, Alerts) {
+
+    // Define share properties through the application controllers (view-moddel).
```
Comment (amitaibu) [inline client/app/scripts/controllers/dashboard.js]: moddel => model

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**